website/docs/docker.md

```diff
@@ -93,7 +93,7 @@ Add `"default:automergeDigest"` to your `extends` array. Also add `"default:auto
 
 ##### Registry authentication
 
-If you are running your own Renovate bot, add this to your `config.js`:
+If you are running your own docker registry, add this to your `config.js`:
```

Comment:
The reason for saying "your own Renovate bot" is because it's only valid for self-hosting, while this page in general is valid for both self-hosted as well as the hosted app.